youtube_dl/extractor/skillshare.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import unicode_literals
+
+from datetime import datetime
+import json
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import ExtractorError
+from ..utils import int_or_none
+
+
+class SkillshareBaseIE(InfoExtractor):
```

Comment:
There is no point in base class.

Reply:
My thought on the base class was to make it simpler to allow more features such single video / playlist downloading later on. @dstftw